AnalysisCode

```diff
@@ -0,0 +1,12 @@
+import csv
+
+NULL = 0
+REPOLIST_CSV = '/Users/medapa/Dropbox/HEC/Summer Project/Data/RepoList2014.csv'
+EVENTLIST_CSV = '/Users/medapa/Dropbox/HEC/Summer Project/Data/EventList2014.csv'
+
+
+def main()
```

Comment:
Lets write the code for calculating deg of superposition and total number of contributors here